internal/cli/atlas/dbusers/certs/list.go

```diff
@@ -59,6 +59,10 @@ func ListBuilder() *cobra.Command {
 		Aliases: []string{"ls"},
 		Short:   "List of all Atlas-managed, unexpired certificates for a database user.",
 		Args:    require.ExactArgs(1),
+		Annotations: map[string]string{
+			"args":         "username",
+			"usernameDesc": "Username of the database user to get certificates for.",
```

Comment:
[nit] Ending in a for doesn't read well here.

```suggestion
			"usernameDesc": "Username of the database user for which you require certificates.",
```